proto/trtl/pb/v1/trtl.proto

```diff
@@ -29,7 +29,8 @@ service Trtl {
 
 message GetRequest {
     bytes key = 1;
-    Options options = 2;
+    bytes namespace = 2;
```

Comment:
Having the `namespace` be a `bytes` object makes sense for parity with the key and for making key operations such as prepend easier. Note, however that we've identified the namespace in the [honu code as a string](https://github.com/rotationalio/honu/blob/main/proto/object/v1/object.proto#L21). We'll either have to change that field in Honu or convert the bytes into a string before saving it. 

The difference between `string` and `bytes` in code is that `string` is much more restrictive about what bytes can be included in the slice; namely a string can only contain "printable" bytes via the UTF-8 specification, whereas the byte slice can contain any byte value. Because strings in Go are UTF-8, it is fairly straight forward to cast a `string` to `[]byte` and vice versa. However, slices cannot be used as keys in a `map` since slices do not have equality defined. I recommend we choose our namespace type based on what is easiest for both the user and for us in the code.